README.md

```diff
@@ -19,8 +19,8 @@ In order to run this API locally you will need to install the following:
 | `E5_USERNAME`                    |   `-`   | E5 API Username                                                       |
 | `BIND_ADDR`                      |   `-`   | The host:port to bind to                                              |
 | `MONGODB_URL`                    |   `-`   | The mongo db connection string                                        |
-| `LFP_MONGODB_DATABASE`           |   `-`   | The database name to connect to e.g. `late_filing_penalties`          |
-| `LFP_MONGODB_COLLECTION`         |   `-`   | The collection name e.g. `payable_resources`                          |
+| `PPS_MONGODB_DATABASE`           |   `-`   | The database name to connect to e.g. `payment_penalties`          |
```

Comment:
I don't think we can change the database connection just yet?
So this should probably remain as `late_filing_penalties`.

Reply:
Yeah, will update.